sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentSentiment_internal.cs

```diff
@@ -5,12 +5,8 @@
 
 namespace Azure.AI.TextAnalytics.Models
 {
-    [CodeGenModel("DocumentSentimentValue")]
-    internal enum DocumentSentimentValue
+    [CodeGenModel("DocumentSentiment")]
+    internal partial struct DocumentSentiment_internal
```

Comment:
`_internal` isn't a very common naming convention for C# types.  I usually see just `Internal` like `DocumentSentimentInternal`.

Reply:
interesting. I will rename this and I created https://github.com/Azure/azure-sdk-for-net/issues/14312 to do renames to other types in the library to not pollute this PR